includes/base_controls/QCheckBox.class.php

```diff
@@ -146,7 +146,10 @@ protected function renderLabelAttributes() {
 			if (!$this->Enabled) {
 				$objStyler->AddCssClass('disabled');	// add the disabled class to the label for styling
 			}
-			return $objStyler->renderHtmlAttributes();
+			if (!$this->Display) {
```

Comment:
Not in this case. If the user puts a display parameter on the $objStyler to turn it off or do something else, then it will override that and turn it to the default on. I only want to override the display if it is specifically turned off.